### PR TITLE
Update entrypoint.sh

### LINKED ISSUE #3: Getting "fatal" result
Hi, thanks for your useful action for *.pot files generation!

I tried to use your action yesterday to generate a pot file for a plugin. I applied the same settings as before for the previous plugin. But the file after generation is not written to my repository. Is this something on your side or should I change the launch of the action in my yml file?

There are the logs for the previous using (2022-02-07) and yesterday (22-04-17) 
[log 2022-04-17.txt](https://github.com/iamdharmesh/action-wordpress-pot-generator/files/8504618/log.2022-04-17.txt)
[log 2022-02-07.txt](https://github.com/iamdharmesh/action-wordpress-pot-generator/files/8504619/log.2022-02-07.txt)

Best Regards!

### PR DESCRIPTION
**Pull request by odil-io**
Because of [this error in Checkout](https://github.com/actions/checkout/issues/766)
I've made a temporary solution to the problem described in #3 using [labeneko's](https://github.com/gaurav-nelson/github-action-markdown-link-check/issues/132#issuecomment-1098020617) suggestion.

### CODE REVIEW

File: entrypoint.sh [hunk 1]
```diff
@@ -38,6 +38,7 @@ fi
 # Setup Git config and push .pot file to github repo
 git config --global user.name "WordPress .pot File Generator"
 git config --global user.email "41898282+github-actions[bot]@users.noreply.github.com"
+git config --global --add safe.directory /github/workspace
```
Comment (iamdharmesh) [inline entrypoint.sh]: @odil-io fix looks good to me, but I think using `"$GITHUB_WORKSPACE"` here is a more future-proof solution. what do you think?

`git config --global --add safe.directory "$GITHUB_WORKSPACE"`

**Review by iamdharmesh — CHANGES_REQUESTED** [review]
Thanks a lot for the PR @odil-io, I have added a minor comment on changes, would you like to update it? 

Thanks.

File: entrypoint.sh (hunk 1 above)
Comment (odil-io) [reply entrypoint.sh]: Haven't tried that one yet, it should work right out of the box. I'll give it a go today

Comment (odil-io) [reply entrypoint.sh]: Couldn't help myself and tested it right away. Works as expected!

**Review by iamdharmesh — APPROVED** [review]
LGTM! Thanks a lot for the fix this :+1:

**Pull request merged**